src/map/mob.cpp

```diff
@@ -2629,10 +2629,12 @@ int mob_dead(struct mob_data *md, struct block_list *src, int type)
 		}
 	}
 
+	map_data *mapdata = map_getmapdata(m);
```

Comment:
Not sure, but all the changes in this file look like unrelated misc fixes/cleanups?